headers/functions/arm9.h

```diff
@@ -179,8 +182,8 @@ enum move_id GetItemMoveId(enum item_id item_id);
 bool TestItemAiFlag(enum item_id item_id, int bit_id);
 bool IsItemInTimeDarkness(enum item_id item_id);
 bool IsItemValidVeneer(enum item_id item_id);
-void SetGold(int val);
-int GetGold(void);
+void SetActiveInventory(int inventory);
```

Comment:
Does `enum team_id` fit here for the parameter?

Reply:
Yes, that is a better fit.